database/Reseed/reseed.js

```diff
@@ -91,49 +102,111 @@ const makeNearbyExperience = function (attractionId) {
   return newNearbyExperience;
 };
 
-let nearFile = fs.createWriteStream('nearby9.txt', {flags: 'a'})
-    .on('error', (err) => console.error(err.message));
-let experienceFile = fs.createWriteStream('experience9.txt', {flags: 'a'})
-    .on('error', (err) => console.error(err.message));
-// let restFile = fs.createWriteStream('rest9.txt', {flags: 'a'})
-//     .on('error', (err) => console.error(err.message));
-// let attractionFile = fs.createWriteStream('attraction9.txt', {flags: 'a'})
-//     .on('error', (err) => console.error(err.message));
-
-attractionIds.forEach((attractionId) => {
-  const lat = chance.latitude({ fixed: 3 });
-  const lng = chance.longitude({ fixed: 3 });
-  const location = {
-    lat,
-    lng
-  };
-  const address = String(chance.address({ short_suffix: true }));
-  const website = chance.email();
-  const phonenumber = chance.phone();
-  const email = chance.email();
-  const nearByRestaurants = [];
-  const nearByAttractions = [];
-  for (let i = 0; i < 3; i += 1) {
-    nearByRestaurants.push(makeNearbyRestaurant(i, location, attractionId));
-    nearByAttractions.push(makeNearbyAttraction(i, location, attractionId));
+async function * generate() {
```

Comment:
I really like this way of generation - I am going to try it for the Arrango! nice work.